### PR TITLE
Remove Large File Copies in Notebook Aux Files

### PR DESCRIPTION
**Pull request by alibillalhammoud**
This update removes several MBs from the aux directory for notebooks by removing redundant files.

### CODE REVIEW

**Comment by msaligane** [comment]
@luccareinehr Can you please review this. It sounds like your last PR broke the notebook. We need to find a way to avoid that in the future.

File: docs/source/notebooks/aux_files/klayout.lyt [hunk 1]
```diff
@@ -71,7 +71,7 @@
    <macro-resolution-mode>default</macro-resolution-mode>
    <separate-groups>false</separate-groups>
    <map-file/>
-   <lef-files>/content/OpenFASOC/docs/source/notebookFiles/klayout_tech.lef</lef-files> <lef-files>/content/OpenFASOC/openfasoc/common/platforms/sky130hd/lef/sky130_fd_sc_hd_merged.lef</lef-files> <lef-files>/content/OpenFASOC/openfasoc/generators/temp-sense-gen/blocks/sky130hd/lef/HEADER.lef</lef-files> <lef-files>/content/OpenFASOC/openfasoc/generators/temp-sense-gen/blocks/sky130hd/lef/SLC.lef</lef-files>
+   <lef-files>/content/OpenFASOC/docs/source/notebooks/aux_files/klayout_tech.lef</lef-files> <lef-files>/content/OpenFASOC/openfasoc/common/platforms/sky130hd/lef/sky130_fd_sc_hd_merged.lef</lef-files> <lef-files>/content/OpenFASOC/openfasoc/generators/temp-sense-gen/blocks/sky130hd/lef/HEADER.lef</lef-files> <lef-files>/content/OpenFASOC/openfasoc/generators/temp-sense-gen/blocks/sky130hd/lef/SLC.lef</lef-files>
```
Comment (luccareinehr) [inline docs/source/notebooks/aux_files/klayout.lyt:74]: Good, this fixes what my PR broke.

**Comment by luccareinehr** [comment]
@msaligane It looks to be working now, I missed a file in my PR but Ali fixed it here.

Just a note: if you try running the fork's notebook right now (https://colab.research.google.com/github/alibillalhammoud/OpenFASOC/blob/main/docs/source/notebooks/temp_sense_genCollab.ipynb), it won't work because the code clones the original OpenFASOC repo, which doesn't have the change I commented above yet. But once it's merged, it should work without errors.

**Comment by luccareinehr** [comment]
Also, I'm sorry because I misunderstood the notebooks concept. I thought they were intended to be run locally, but running it in Colab is just so much better. That's why I suggested some of the changes in #103.

Maybe we could add a description in [examples.rst](https://github.com/idea-fasoc/OpenFASOC/blob/main/docs/source/examples.rst) saying that the notebooks are supposed to be opened in Google Colab? Though they can be run locally, it creates loads of files in the user's already cloned repo

**Review by luccareinehr — APPROVED** [review]

**Comment by alibillalhammoud** [comment]
@luccareinehr I added a note to examples.rst as you suggested.

**Comment by saicharan0112** [comment]
@luccareinehr @alibillalhammoud Guys try not changing the link of the main notebook since it is been shared with others. Also, if anything such thing is updated, make sure that it is updated inside the notebook too (the colab link). Looks good as of now but something to remember for the future.

**Review by msaligane — APPROVED** [review]

**Pull request merged**